Documentation/contributing/development/reviewers_committers/review_process.rst

```diff
@@ -0,0 +1,168 @@
+   | ``release-note/ci``               | This is a CI feature or bug fix.                                                                       |
+   +-----------------------------------+--------------------------------------------------------------------------------------------------------+
+
+#. Check for upgrade compatibility impact and if in doubt, set the label
+   ``upgrade-impact`` and discuss in the Slack channel or in the weekly
+   meeting.
+
+   +--------------------------+---------------------------------------------------------------------------+
+   | Labels                   | When to set                                                               |
+   +==========================+===========================================================================+
+   | ``upgrade-impact``       | The code changes have a potential upgrade impact                          |
+   +--------------------------+---------------------------------------------------------------------------+
+
+#. When all review objectives for all ``CODEOWNERS`` are met, all CI tests have
+   passed, and all reviewers have approved the requested changes, you can merge
+   the PR by clicking on the "Rebase and merge" button.
+
+#. If the PR is a backport PR, update the labels of cherry-picked PRs with the
+   command included at the end of the original post. For example:
+
+   .. code-block:: shell-session
+
+       $ for pr in 26759 24099; do contrib/backporting/set-labels.py $pr done 1.13; done
+
+Reviewer Teams
+--------------
+
+Every committer in the `committers team`_ belongs to `one or more other teams
+in the Cilium organization <cilium_teams_>`_ If you would like to be added or
+removed from any team, please contact any of the `maintainers`_.
+
+Once a PR is opened by a contributor, GitHub will automatically pick which
+`teams <cilium_teams>`_ should review the PR using the ``CODEOWNERS`` file.
+Each committer can see the PRs they need to review by filtering by reviews
+requested. A good filter is provided in this `link <user_review_filter_>`_ so
+make sure to bookmark it.
+
+Reviewers are expected to focus their review on the areas of the code where
+GitHub requested their review. For small PRs, it may make sense to simply
+review the entire PR. However, if the PR is quite large then it can help to
+narrow the area of focus to one particular aspect of the code. When leaving a
+review, share which areas you focused on and which areas you think that other
+reviewers should look into. This will help others to focus on aspects of review
+that have not been covered as deeply.
+
+Belonging to a team does not mean that a committer should know every single
+line of code the team is maintaining. For this reason it is recommended that
+once you have reviewed a PR, if you feel that another pair of eyes is needed,
+you should re-request a review from the appropriate team. In the example below,
```

Comment:
```suggestion
it's okay to re-request a review from the appropriate team. In the following example,
```

Reply:
Actually... Let's just instruct to re-request a review, “it's okay to” is not necessary. We do want more reviews if the reviewers are unsure.